docs/source/usersguide/geometry.rst

```diff
@@ -183,9 +183,13 @@ the :class:`openmc.Cell` class::
   fuel.fill = uo2
   fuel.region = pellet
 
+  # This cell will be filled with void on export to XML
+  gap = openmc.Cell(region=pellet_gap)
+
 In this example, an instance of :class:`openmc.Material` is assigned to the
 :attr:`Cell.fill` attribute. One can also fill a cell with a :ref:`universe
-<usersguide_universes>` or :ref:`lattice <usersguide_lattices>`.
+<usersguide_universes>` or :ref:`lattice <usersguide_lattices>`. If no fill 
+is provided to a cell, it will be filled with void by default.
```

Comment:
```suggestion
<usersguide_universes>` or :ref:`lattice <usersguide_lattices>`. If you provide
no fill to a cell or assign a value of `None`, it will be treated as a "void"
cell with no material within. Particles are allowed to stream through the cell but
will undergo no collisions::

  # This cell will be filled with void on export to XML
  gap = openmc.Cell(region=pellet_gap)
```

Reply:
So i messed up and moved the line before I noticed that you have suggested the change. I applied the suggestion in Commit 9c8e22add86b84f334999e7a160dbbe0ad1caa9c. Sorry for the mess.